pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3314,3 +3361,18 @@ func (c *Cloud) addFilters(filters []*ec2.Filter) []*ec2.Filter {
 func (c *Cloud) getClusterName() string {
 	return c.filterTags[TagNameKubernetesCluster]
 }
+
+func setNodeDisk(
```

Comment:
We are using this function in 2 place @justinsb , so if we inline this we will have to copy same 3-4 lines in 2 places. perhaps lets leave it like how it is?